modules/sdk/project-templates/project-templates-workspace/src/main/resources/archetype-resources/settings.gradle

```diff
@@ -10,6 +10,7 @@ buildscript {
 	repositories {
 		maven {
 			url "https://repository-cdn.liferay.com/nexus/content/groups/public"
+			url "https://repository.liferay.com/nexus/content/groups/public"
```

Comment:
Does this work like this?  Do you need to have a second `maven { url ... }` block just like we did up in the code?